specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2017-10-01/swagger.json

```diff
@@ -0,0 +1,335 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-10-01",
+    "title": "Azure Log Analytics",
+    "description": "This API exposes Azure Log Analytics query capabilities",
+    "termsOfService": "https://dev.loganalytics.io/tos",
+    "contact": {
+      "name": "AIAPI Team",
+      "url": "https://dev.loganalytics.io/support",
+      "email": "aiapi@microsoft.com"
+    },
+    "license": {
+      "name": "Microsoft",
+      "url": "https://dev.loganalytics.io/license"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/query": {
+      "post": {
+        "operationId": "Query_Post",
```

Comment:
Same comment as in the other API Version, I'd imagine this should be `Query_Create`.